src/Colors.stories.mdx

```diff
@@ -0,0 +1,49 @@
+<!-- Colors.stories.mdx -->
+
+import { Meta, ColorPalette, ColorItem } from '@storybook/addon-docs';
+import './scss/colors.scss';
+
+<Meta title="Colors" />
+
+<ColorPalette>
+  <ColorItem title="Primary" subtitle="LighterBordeaux" colors={{ Background: '#eee8ea' }} />
+  <ColorItem
+    title="Accent"
+    subtitle="Bordeaux"
+    colors={{ Accent: '#591D2D', AccentHover: '#73263A' }}
+  />
+  <ColorItem
+    title="Text"
+    subtitle="Gray scale and bordeaux gradient"
+    colors={{
+      Text: '#4d4d4d',
+      TextLight: '#666666',
```

Comment:
The casing for the displayed color name should be the same as for the `scss` variables, i.e. `'text-light': '#666666'`